wikia-robots-txt.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+if ( !defined( 'MW_NO_SETUP' ) ) {
+	define( 'MW_NO_SETUP', 1 );
+}
+require_once( __DIR__ . '/includes/WebStart.php' );
+require_once( __DIR__ . '/includes/Setup.php' );
+
+if ( empty( $wgEnableRobotsTxtExt ) ) {
+	exit;
+}
+
+$robots = new RobotsTxt();
+
+if ( !empty( $_SERVER['HTTP_X_STAGING'] ) ) {
```

Comment:
should probably block devboxes here too (just to be safe, i know that's currently done another way now)